src/test/java/com/faforever/client/preferences/PreferencesBuilder.java

```diff
@@ -0,0 +1,660 @@
+package com.faforever.client.preferences;
+
+import com.faforever.client.chat.ChatColorMode;
+import com.faforever.client.chat.ChatFormat;
+import com.faforever.client.chat.ChatUserCategory;
+import com.faforever.client.game.Faction;
+import com.faforever.client.game.GamesTilesContainerController.TilesSortingOrder;
+import com.faforever.client.main.event.NavigationItem;
+import com.faforever.client.map.generator.GenerationType;
+import com.faforever.client.preferences.Preferences.UnitDataBaseType;
+import com.faforever.client.vault.search.SearchController.SortConfig;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.collections.ObservableMap;
+import javafx.scene.control.TableColumn.SortType;
+import javafx.scene.paint.Color;
+import javafx.util.Pair;
+
+import java.net.HttpCookie;
+import java.net.URI;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+public class PreferencesBuilder {
+
+  private final Preferences preferences;
+
+  private PreferencesBuilder() {
+    preferences = new Preferences();
+  }
+
+  public static PreferencesBuilder create() {
+    return new PreferencesBuilder();
+  }
+
+  public PreferencesBuilder defaultValues() {
+    return this;
+  }
```

Comment:
Where are the default values coming from?

Reply:
The default values are set the ones set when the preferences object is created although in that case it may be better to just remove the default values